### PR TITLE
Feat/support create

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: packages/manager/modules/support/src/new-ticket/creation-form/creation-form.controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export default class SupportNewCreationFormController {
+  /* @ngInject */
+  constructor($q, $state, $translate, OvhApiService, OvhApiSupport) {
+    this.$q = $q;
+    this.$state = $state;
+    this.$translate = $translate;
+    this.OvhApiService = OvhApiService;
+    this.OvhApiSupport = OvhApiSupport;
+  }
+
```
Comment (FredericEspiau) [inline packages/manager/modules/support/src/new-ticket/creation-form/creation-form.controller.js]:
````
$onInit() {
    this.subject = get(
        findLast(this.issues, 'subject'),
        'subject'
    )
}
````

File: packages/manager/apps/support/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "font-awesome": "^4.7.0",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
-    "ovh-api-services": "^9.5.1",
+    "ovh-api-services": "^9.7.0",
```
Comment (marie-j) [inline packages/manager/apps/support/package.json]: You should probably rebase with `develop` as verison is now `^9.8.0`

File: packages/manager/modules/support/src/new-ticket/creation-form/creation-form.controller.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import findLast from 'lodash/findLast';
+
+export default class SupportNewCreationFormController {
+  /* @ngInject */
+  constructor($q, $state, $translate, OvhApiService, OvhApiSupport) {
+    this.$q = $q;
+    this.$state = $state;
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/creation-form/creation-form.controller.js]: Could this be passed through bindings  ? (also doesn't seem to be used)

File: packages/manager/modules/support/src/new-ticket/creation-form/creation-form.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <input type="text"
+           id="customSubject"
+           name="customSubject"
+           data-required="true"
+           data-ng-model="$ctrl.customSubject"
+           class="oui-input" />
+  </oui-field>
+  
+  <div data-ng-repeat="issue in $ctrl.issues">
+    <oui-field data-ng-repeat="field in issue.fields"
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/creation-form/creation-form.html]:
```suggestion
    <oui-field data-ng-repeat="field in issue.fields track by field.id"
```
maybe ?

File: packages/manager/modules/support/src/new-ticket/issues-form/issues-form.controller.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  }
+
+  $onInit() {
+    this.loading = true;
+    return this.$q.all({
+      schema: this.OvhApiSupport.v6().schema().$promise,
+      serviceTypes: this.OvhApiSupport.v6().getServiceTypes().$promise,
+      supportLevel : this.OvhApiMe.v6().supportLevel().$promise,
+    }).then(({ schema, serviceTypes, supportLevel }) => {
+      this.categories = schema.models['support.tickets.CategoryEnum'].enum.map(categoryId => ({
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/issues-form/issues-form.controller.js]: constant for `support.tickets.CategoryEnum` maybe ?

File: packages/manager/modules/support/src/new-ticket/issues-selector/issues-selector.controller.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    return this.serviceTypeValue;
+  }
+
+  set serviceType(value) {
+    this.issueType = null;
+    this.serviceTypeValue = value;
+    this.fetchIssueTypes();
+  }
+
+  get issuesHierarchy() {
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/issues-selector/issues-selector.controller.js]: Seems like a `Issue` model could be used maybe

File: packages/manager/modules/support/src/new-ticket/new-ticket.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
+<oui-back-button data-state="support.tickets">
+  {{ 'ovhManagerSupport_new_back' | translate }}
+</oui-back-button>
+
+<support-new-issues-form
+  data-ng-show="$ctrl.step === 'issues'"
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/new-ticket.html]:
```suggestion
  data-ng-if="$ctrl.step === 'issues'"
```
can't do ?

File: packages/manager/modules/support/src/new-ticket/new-ticket.module.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    angularTranslate,
+    ngTranslateAsyncLoader,
+    'oui',
+    uiRouter,
+  ])
+  .component(component.name, component)
+  .component(creationFormComponent.name, creationFormComponent)
+  .component(issuesSelectorComponent.name, issuesSelectorComponent)
+  .component(issuesFormComponent.name, issuesFormComponent);
+//  .config(/* @ngInject */ ($stateProvider) => {
```
Comment (marie-j) [inline packages/manager/modules/support/src/new-ticket/new-ticket.module.js]: shoud be removed if not necessary

**Pull request merged**